lib/livebook/file_system/s3.ex

```diff
@@ -14,6 +14,12 @@ defmodule Livebook.FileSystem.S3 do
           hub_id: String.t()
         }
 
+  @type credentials_t :: %{
+          access_key_id: String.t() | nil,
+          secret_access_key: String.t() | nil,
+          token: String.t() | nil
```

Comment:
If this is the session token, then I think being more explicit is definitely helpful

```suggestion
          session_token: String.t() | nil
```

Reply:
I changed it to `token` because that's what `aws_credentials` returned in its map, but I was on the fence about it.